### ISSUE TITLE
How to load DatabaseVariantAnnotation information ?

### ISSUE THREAD

**Issue by xubo245** [bigdatagenomics/adam#1049]
I have tried :  
data: small.vcf  in adam-core test resources
methods:
1.  val rdd = sc.loadVcfAnnotations(path)
2.    val rdd = sc.loadVariantAnnotations(path)
3. val rdd = sc.loadVcf(path, sd)
   val rddDatabase = rdd.flatMap(_.databases)
   4.loadParquetVariantAnnotations
   
   val path = "xubo/callVariant/vcf/small.vcf"  or adam format

these methods can not get dbSnpid

Is data error? or my used metheds error ? or now adam do not support ?

**Comment by fnothaft** [comment]
Should be resolved by #1250.

**Comment by heuermh** [comment]
Fixed by #1288.

Annotations are now provided for Variants by default (unless projected away).  The VCF ID column maps to the `Variant.names` field and the DB VCF INFO flag maps to the `VariantAnnotation.dbSnp` field.

In `adam-shell`
```
scala> import org.bdgenomics.adam.rdd.ADAMContext._
import org.bdgenomics.adam.rdd.ADAMContext._

scala> val variants = sc.loadVariants("foo.vcf")
variants: org.bdgenomics.adam.rdd.variant.VariantRDD = ...

scala> val head = variants.rdd.collect.head
head: org.bdgenomics.formats.avro.Variant = {"contigName": "1", "start": 14396, "end": ...

scala> head.names
res0: java.util.List[String] = ["rs12076"]

scala> head.getAnnotation.dbSnp
res1: Boolean = true
```
